### ISSUE TITLE
Riot stopped working

### ISSUE THREAD

**Issue by aladar42** [vector-im/element-web#8313]
### Description

I am quite at a loss for what is going on. Riot crashed for me today (or, rather, the Electron portion of it crashed, the process just displayed an empty white window), and after restarting it, it wouldnt connect to my account. It just hangs on the loggin in screen infinitely, after throwing an "Unable to connect to homeserver" error. I tried reinstalling the client, deleting the config, running Riot in the terminal and looking for any errors... All in vain. Other clients and Riot for Android connect just fine.

### Steps to reproduce

- Run the software

Describe how what happens differs from what you expected.

Log: Not sent, software stuck in infinite loop, even "log out" button is unresponsive

### Version information

- **Platform**: web (in-browser) or desktop?

Desktop

For the desktop app:

- **OS**: Arch Linux
- **Version**: 0.17.9

**Comment by aaronraimist** [comment]
Can you open the console and paste that in this issue? 

Keyboard shortcut is Control+Shift+I

Also what homeserver are you on?

**Comment by aladar42** [comment]
Is this what you mean? https://pastebin.com/31dmKt5x

Im on disroot.org. Every other client I tried (Riot for Android, Fractal, nheko) work just fine, even with config file deleted and starting from fresh.

**Comment by aaronraimist** [comment]
Yes thanks, that'll be required for someone who knows more than me to be able to debug it.

I don't know a whole lot but as far as I can tell it is working but the server is timing out, maybe you just need to leave it sitting open a while longer.

**Comment by aladar42** [comment]
Ive had it running for hours, its just stuck in this loop infinitely. Every other client works, so I dont think its the server - even the homeserver-hosted Riot webclient works fine.

**Comment by floviolleau** [comment]
Any news?

**Comment by turt2live** [comment]
As far as I can tell, the homeserver is timing out and that is causing problems.

**Comment by aladar42** [comment]
I can log in just fine with any other client, including Riot for Android, and the HS Riot webhost, so that's probably not the case.

**Comment by t3chguy** [comment]
Are you still experiencing this?

**Comment by weeman1337** [comment]
I am closing this issue because there has been no response in the last two weeks. If you keep experiencing this defect, please add a comment with @weeman1337 to let me know that it should be reopened.
